src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachExceptionRecordToExistingCaseTest.java

```diff
@@ -551,6 +552,50 @@ public void should_succeed_when_classification_is_exception_and_exception_record
         verifyRequestedAttachingToCase();
     }
 
+    @Test
+    public void should_succeed_when_classification_is_supplementary_evidence_with_ocr() {
+        CallbackRequest callbackRequest =
+            callbackRequestWith(
+                EVENT_ID_ATTACH_TO_CASE,
+                SUPPLEMENTARY_EVIDENCE_WITH_OCR.name(),
+                true
+            );
+
+        ValidatableResponse response =
+            given()
+                .body(callbackRequest)
+                .headers(userHeaders())
+                .post(CALLBACK_ATTACH_CASE_PATH)
+                .then()
+                .statusCode(200);
+
+        verifySuccessResponse(response, callbackRequest);
+        verifyRequestedAttachingToCase();
+    }
+
+    @Test
+    public void should_fail_when_classification_is_supplementary_evidence_with_ocr_does_not_include_ocr() {
+        CallbackRequest callbackRequest =
+            callbackRequestWith(
+                EVENT_ID_ATTACH_TO_CASE,
+                SUPPLEMENTARY_EVIDENCE_WITH_OCR.name(),
+                false
+            );
+
+        ValidatableResponse response =
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Found 'DU'-anomaly for variable 'response' (lines '585'-'597').](https://app.codacy.com/app/HMCTS/bulk-scan-orchestrator/pullRequest?prid=4550029)